schema_registry.go

```diff
@@ -103,3 +109,30 @@ func CreateSchema(
 	}
 	return schemaInfo, nil
 }
+
+func GetSubjectName(schema string, topic string, element Element, subjectNameStrategy string) (string, *Xk6KafkaError) {
+	if subjectNameStrategy == "" || subjectNameStrategy == "TopicNameStrategy" {
+		return topic + "-" + string(element), nil
+	}
+
+	var schemaMap map[string]interface{}
+	err := json.Unmarshal([]byte(schema), &schemaMap)
+	if err != nil {
+		return "", NewXk6KafkaError(failedEncodeToAvro, "Schema invalid json", nil)
+	}
+	var recordName = ""
+	if namespace, ok := schemaMap["namespace"]; ok {
+		recordName = namespace.(string) + "."
+	}
+	recordName += schemaMap["name"].(string)
+
+	if subjectNameStrategy == "RecordNameStrategy" {
+		return recordName, nil
+	}
+	if subjectNameStrategy == "TopicRecordNameStrategy" {
+		return topic + "-" + recordName, nil
+	}
```

Comment:
Make the `"...NameStrategy"` literals as module-level constants, and I'll eventually export them:
- #70